mlflow/models/dependencies_schema.py

```diff
@@ -0,0 +1,160 @@
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Dict, List, Optional
+
+from mlflow.utils.annotations import experimental
+
+DATABRICKS_VECTOR_SEARCH_PRIMARY_KEY = "__databricks_vector_search_primary_key__"
+DATABRICKS_VECTOR_SEARCH_TEXT_COLUMN = "__databricks_vector_search_text_column__"
+DATABRICKS_VECTOR_SEARCH_DOC_URI = "__databricks_vector_search_doc_uri__"
+DATABRICKS_VECTOR_SEARCH_OTHER_COLUMNS = "__databricks_vector_search_other_columns__"
+
+
+@experimental
+def set_vector_search_schema(
+    primary_key: str,
+    text_column: str = "",
```

Comment:
Is empty string a valid vector search schema column name, or are we using this to represent an empty value? If we're using this to represent an empty value, we should use `None` instead.